hack/charts/hyperparameter-tuning-jobs/Chart.yaml

```diff
@@ -1,9 +1,13 @@
 apiVersion: v1
 name: sagemaker-k8s-trainingjob
```

Comment:
I think this name should change to `sagemaker-k8s-hyperparametertuningjob` or perhaps `sagemaker-k8s-hpo`